.github/workflows/fossa.yml

```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13.1'
+    - run: go version
+    # Runs a set of commands to initialize and analyze with FOSSA
+    - name: Install and run FOSSA analysis
```

Comment:
Might be worth to add fossa-cli to our build tools image: https://github.com/kedacore/keda/blob/389ea136d669e2f46cbc09b222d3255ebb0361aa/tools/build-tools.Dockerfile and here just run the analyze.
This way we have all tools consistently installed in one image.
WDYT?

Reply:
@zroubalik np, let's do it :)